rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc

```diff
@@ -27,9 +27,20 @@ Machine pools are a higher level construct to compute machine sets.
 
 A machine pool creates compute machine sets that are all clones of the same configuration across availability zones. Machine pools perform all of the host node provisioning management actions on a worker node. If you need more machines or must scale them down, change the number of replicas in the machine pool to meet your compute needs. You can manually configure scaling or set autoscaling.
 
-By default, a cluster is created with one machine pool. You can add additional machine pools to an existing cluster, modify the default machine pool, and delete machine pools.
+By default, a cluster is installed with one machine pool. You can label that machine pool during installation. The `Default` machine pool node type or size cannot be changed after the cluster is installed, and that machine pool cannot be deleted. 
```

Comment:
Passive to active. Suggestion:

> By default, the installation program installs a cluster with one machine pool. During cluster installation, you can label this machine pool. 

After the installation program installs the cluster, you cannot modify the `Defualt` machine pool's node type or size or delete the machine pool.

Reply:
Again, I don't mind the passive here. Stating that _the installation program installs a cluster_ is certainly accurate, but it seems unnecessary to me. 
Maybe:
> By default, a cluster **_has_** one machine pool.

Reply:
Maybe:
By default, a cluster has one machine pool. During cluster installation, you can label this machine pool. After the cluster is installed, the `Default` machine pool cannot be deleted and its node type or size cannot be changed.